src/javascript/app_2/App/Components/Layout/Footer/__tests__/toggle_settings.spec.js

```diff
@@ -0,0 +1,61 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import { ToggleSettings }     from '../toggle_settings.jsx';
+import { IconSettings }       from 'Assets/Footer';
+import SettingsDialog         from '../../../Elements/SettingsDialog/settings_dialog.jsx';
+import { CSSTransition }      from 'react-transition-group';
+
+configure({ adapter: new Adapter() });
+
+describe('ToggleSettings', () => {
+    it('should render one <ToggleSettings /> component', () => {
+        const wrapper = shallow(<ToggleSettings  />);
+        expect(wrapper).to.have.length(1);
+    });
+    it('should have active class if is_settings_visible is true', () => {
```

Comment:
Suggestion: Merge these 2 tests inside 1 unit, basically we expect `.active` to be activated based on `is_settings_visible` value. Just for better readability in the output.